master/internal/api_tensorboard.go

```diff
@@ -477,10 +477,14 @@ func (a *apiServer) getTensorBoardConfigsFromReq(
 			return nil, err
 		}
 
-		conf, err := db.LegacyExperimentConfigByID(int(expID))
```

Comment:
renaming this and coupling it to the unmanaged feels strange to me

I think ```getExperimentAndCheckCanDoActions``` returns the experiment as the first parameter andthat experiment struct should have ```e.Unmanaged```?